app/controllers/admin_controller.rb

```diff
@@ -4,23 +4,17 @@ def jobs
   end
 
   def seeds
-    #@seeds = SeedUri.all
-    #@seeds =SeedUri.joins("LEFT OUTER JOIN mementos On mementos.uri_id = seed_uris.id").group("seed_uris.id","seed_uris.uri").select("seed_uris.id","seed_uris.uri","mementos.is_thumbnail_captured").sum("is_thumbnail_captured")
-     @seeds =SeedUri.joins("LEFT OUTER JOIN (select sum(mementos.is_thumbnail_captured) as captured, count(mementos.is_selected) as no_mementos, mementos.uri_id from mementos group by uri_id) as"+
+    @seeds =SeedUri.joins("LEFT OUTER JOIN (select sum(mementos.is_thumbnail_captured) as captured, count(mementos.is_selected) as no_mementos, mementos.uri_id from mementos group by uri_id) as"+
```

Comment:
I'm a bit concerned that this SQL lives in the controller. Maybe it could live within the model in a method? Also there seems to be some inconsistent spacing with the `=` operator